### DISCUSSION TITLE
Features flags to be returned from the local file

### DISCUSSION THREAD

**Discussion by prabuanandan** [Flagsmith/flagsmith#2313] [General]
Hello All,

   I am new to flagsmith, I am wondering is there a possibility to return the flags from the local json file rather calling the API. I want to export the result of the /api/v1/flags/ to a json file and block the flagsmith to not call the API rather look into this json file to return the state of the flag? Is this possible to do ?

**Comment by dabeeeenster** [comment]
Hi,

Yes - possibly - which SDK are you using? 

We also have https://github.com/Flagsmith/flagsmith-cli for the JSON...

**Reply by prabuanandan** [reply]
Thanks for responding @dabeeeenster. I am using java sdk. If you can help me with any example or tutorial it would be great.

**Reply by dabeeeenster** [reply]
Not sure if this is a capability of the Java SDK - shout out to @matthewelwell who knows this!
